flow/activities/flowable.go

```diff
@@ -173,20 +173,13 @@ func (a *FlowableActivity) StartFlow(ctx context.Context,
 		return nil, fmt.Errorf("failed to get destination connector: %w", err)
 	}
 	defer connectors.CloseConnector(dstConn)
-
-	log.WithFields(log.Fields{
-		"flowName": input.FlowConnectionConfigs.FlowJobName,
-	}).Infof("initializing table schema...")
+	slog.Info("initializing table schema...", shared.FlowNameKey, ctx.Value(shared.FlowNameKey))
```

Comment:
Seems a bit annoying having to do `x, ctx.Value(x)`

Reply:
can we have a `logging` package?
It would be great to do: `log.with(ctx).Infof(...)`, and the `with` method takes care of adding the additional fields